p9Layouts/defaultPnorLayout_64.xml

```diff
@@ -149,18 +149,18 @@ Layout Description
         <ecc/>
     </section>
     <section>
-        <description>SBE-IPL (Staging Area) (288K)</description>
+        <description>SBE-IPL (Staging Area) (520K)</description>
         <eyeCatch>SBE</eyeCatch>
         <physicalOffset>0xF90000</physicalOffset>
-        <physicalRegionSize>0x48000</physicalRegionSize>
+        <physicalRegionSize>0x82000</physicalRegionSize>
         <sha512perEC/>
         <side>A</side>
         <ecc/>
     </section>
     <section>
         <description>HCODE Ref Image (1.125MB)</description>
         <eyeCatch>HCODE</eyeCatch>
-        <physicalOffset>0xFE0000</physicalOffset>
+        <physicalOffset>0x101A000</physicalOffset>
```

Comment:
I don't understand why we're not computing the math directly here.  From lines 154 and 155 this should be 0x1012000.  Since the number is bigger, I guess it's not a problem, but it does make it harder to review.